scripts/ayp.coffee

```diff
@@ -312,13 +312,17 @@ class AYPStrip
 
     @loadAvatars names, (err, avatars) =>
       return cb(err, null) if err
+      scrambleLines = []
 
       if names.length == 1
         # The only person speaking is centered in the frame
         char = avatars[names[0]]
         left = (frame.width / 2) - (char.width / 2)
         top = (frame.height - char.height)
         @compositeImage frame, char, left, top
+
+        isKenny = char.matchchar.match('default') == null
```

Comment:
> Would your proposal work?

I have no idea :D

How about checking if it ends in `default.png`, or something more specific?